codec/types/any.go

```diff
@@ -64,20 +64,14 @@ func NewAnyWithValue(v proto.Message) (*Any, error) {
 	if v == nil {
 		return nil, sdkerrors.Wrap(sdkerrors.ErrPackAny, "Expecting non nil value to create a new Any")
 	}
-	return NewAnyWithCustomTypeURL(v, "/"+proto.MessageName(v))
-}
 
-// NewAnyWithCustomTypeURL same as NewAnyWithValue, but sets a custom type url, instead
-// using the one from proto.Message.
-// NOTE: This functions should be only used for types with additional logic bundled
-// into the protobuf Any serialization. For simple marshaling you should use NewAnyWithValue.
-func NewAnyWithCustomTypeURL(v proto.Message, typeURL string) (*Any, error) {
```

Comment:
Don't we still need this to support ADR 031 method names?

Reply:
This was only called by the TxBuilder to SetMsg with ServiceMsgs. I removed support for ServiceMsgs on the TxBuilder.

The modules still call `RegisterMsgServiceDesc`, which in turn calls `RegisterCustomTypeURL`, and that's the method needed to support ADR 031 method names.